packages/git/src/node/git-repository-manager.ts

```diff
@@ -30,7 +30,7 @@ export class GitRepositoryManager {
 
     run<T>(repository: Repository, op: () => Promise<T>): Promise<T> {
         const result = op();
-        result.then(() => this.sync(repository));
+        result.then(() => this.sync(repository)).catch(e => console.log(e));
```

Comment:
Only the `sync` is actually unhandled, whereas the `op()` call is handled at a higher level. If we use a catch-all mechanism here, we potentially swallow errors that should be thrown and caught on the caller site. We should only call `catch` on the `sync` promise:
```suggestion
        result.then(() => this.sync(repository).catch(e => console.log(e)));
```